modules/db/src/main/Handlers.scala

```diff
@@ -9,25 +9,31 @@ import scala.util.{ Failure, NotGiven, Success, Try }
 import lila.common.Iso.{ *, given }
 import lila.core.data.Percent
 import lila.core.net.IpAddress
+import lila.core.game.Blurs
 
 trait Handlers:
 
   def toBdoc[A](a: A)(using writer: BSONDocumentWriter[A]): Option[BSONDocument] = writer.writeOpt(a)
 
-  // free handlers for all types with TotalWrapper
+  // free writer for all types with TotalWrapper
   // unless they are given an instance of lila.db.NoDbHandler[T]
   given opaqueWriter[T, A](using
       rs: SameRuntime[T, A],
       writer: BSONWriter[A]
   )(using NotGiven[NoDbHandler[T]]): BSONWriter[T] with
```

Comment:
shouldn't `NoDbHandler` be a super type of `NoBSONReader` then? or renamed `NoBSONWriter`?

Reply:
yeah, it should be renamed to `NoBSONWriter`. We split `opaqueWriter` and `opaqueReader` then `NoBSONReader` and `NoBSONWriter` should be split as well.